apple/internal/templates/macos.template.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2015 The Bazel Authors. All rights reserved.
```

Comment:
I copied this header, let me know if I need to strip it, update the copyright year, etc